apps/dashboard/src/app/(dashboard)/(chain)/components/server/products.ts

```diff
@@ -49,6 +50,13 @@ export const products = [
     description: "Point of sale solution for bridging, onramping & swapping",
     link: "https://portal.thirdweb.com/connect/pay/overview",
   },
+  {
+    name: "Insight",
+    id: "insight",
+    icon: InsightIcon,
+    description: "Query, transform and analyze blockchain data",
+    link: "https://portal.thirdweb.com/insight",
+  },
```

Comment:
In the chainlist page in dashboard, No chains are shown to be supporting insight service - is that correct? 

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/yNOf1svJ8o3zjO7zQouZ/42fd4544-7bb2-4395-b418-95ae608bd938.png)

Reply:
It's querying dev api, which hasn't indexed any chains. If I run it locally against prod api, I do see chains with insight enabled